site/backup.md

```diff
@@ -2,6 +2,11 @@
 
 ## Overview
 
+NB: modern versions of RabbitMQ (3.8.0+) only support backup restore from disk
+when they are restored to a RabbitMQ node with exactly the same node name as they
+node the data was backed up from. Node renaming is not supported if either quorum
+queues or streams are used. It is better to rely on other DR solutions.
```

Comment:
DR-> [disaster recovery](https://blog.rabbitmq.com/posts/2020/07/disaster-recovery-and-high-availability-101/). 

NB: This abbreviation is not defined in this document.